### PR TITLE
Earlytracks-Jobs

### PR DESCRIPTION
**Pull request by Zwordi**
We are looking!!!!

### CODE REVIEW

File: earlytracks-pythonjs-developper-job-advert-title-required.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: Python/JS Developper# Job Advert Title (required)
```
Comment (salimfadhley) [inline earlytracks-pythonjs-developper-job-advert-title-required.html]: You can get rid of the # and every character to the right on each line - they are just comments.

File: earlytracks-pythonjs-developper-job-advert-title-required.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+company: Earlytracks # Your Company (required)
+url: https://www.earlytracks.com/it-dev #<Link to your site/a job spec (optional)>
+location: Louvain-La-Neuve, Belgium # where is the job based? e.g. London,England
+          # Choose one of the following options
+contract: permanent # permanent / contract / part-time / temporary
+contact:
```
Comment (salimfadhley) [inline earlytracks-pythonjs-developper-job-advert-title-required.html]: You have two contract fields, you should only have one.

File: earlytracks-pythonjs-developper-job-advert-title-required.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    name: Sylvain Francis # Your name (required)
+    email: dev@earlytracks.com # Email address applicants should submit to (required)
+    phone: +3210 457 223 # Phone number (optional)
+    # <additional contact fields>: <additional value>
```
Comment (salimfadhley) [inline earlytracks-pythonjs-developper-job-advert-title-required.html]: You can remove the line if you don't ned to use it.

File: earlytracks-pythonjs-developper-job-advert-title-required.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+created: !!timestamp 2018-04-25
+tags:
+  - python
+  - docker
+  - REST
+  - Javascript
+  # include relevant tags
+---
+
+# Title
```
Comment (salimfadhley) [inline earlytracks-pythonjs-developper-job-advert-title-required.html]: You can change the word Title to something more meaningful

File: earlytracks-pythonjs-developper-job-advert-title-required.html [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+We are looking for a Python/JS Developper to join us. They will assist the main developper to some tasks like:
+   
+   - Maintain and develop the main application
+   - Develop a frontend Stack for the SAAS Application
+   - Works on small project for internal use or external (Customer needs)
+   
+We are working to put our dockerize application to a next step and join the SAAS Application World.
```
Comment (salimfadhley) [inline earlytracks-pythonjs-developper-job-advert-title-required.html]: If docker is an important technology you could add that to the list of tags.

File: earlytracks-pythonjs-developper-job-advert-title-required.html [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+We are looking for junior or senior people for this position.
+
+
+In order to achieve this goal we need more people with skills:
```
Comment (salimfadhley) [inline earlytracks-pythonjs-developper-job-advert-title-required.html]: All of these can be added to the tags.

**Comment by Zwordi** [comment]
I made some changes.
Please review.

**Comment by stestagg** [comment]
Apologies for the spam, I was debugging why the preview hadn't worked, and it turns out your job file was in the wrong place,  we should add an automated check for that!
I've moved it, and things are better now.  @salimfadhley, you ok for this to merge?
Steve

**Pull request merged**